x-pack/plugins/synthetics/e2e/journeys/private_locations/add_monitor_private_location.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { journey, step, expect, before } from '@elastic/synthetics';
+import { assertText, byTestId, TIMEOUT_60_SEC } from '@kbn/observability-plugin/e2e/utils';
+import { monitorManagementPageProvider } from '../../page_objects/monitor_management';
+
+journey('AddPrivateLocationMonitor', async ({ page, params: { kibanaUrl } }) => {
+  const uptime = monitorManagementPageProvider({ page, kibanaUrl });
```

Comment:
I would like to see more tests for various permission models. Perhaps we can add them in a follow up PR.